api/server/router/local/container.go

```diff
@@ -396,6 +396,11 @@ func (s *router) postContainersAttach(ctx context.Context, w http.ResponseWriter
 		return derr.ErrorCodeNoSuchContainer.WithArgs(containerName)
 	}
 
+	state := s.daemon.State(containerName)
```

Comment:
@cpuguy83  My first thought is to add `daemon.IsPaused(containerName)`, but I think `daemon.State(containerName)` is much more widely use, in future, maybe we want get other
state of the container, we don't need to add more 'IsXXXX' function